[build] Possible bug in parallel make?

Since a few days we are seeing this strange failure in Coq's CI
https://gitlab.com/coq/coq/-/jobs/152983738
```
"/builds/coq/coq/_install_ci/bin/coqc" -w "+compatibility-notation" -I ../src -R ../theories Equations mutualwfrec.v
File "./ordinal_rec.v", line 3, characters 15-23:
Error: Unable to locate library ordinals.

Makefile:19: recipe for target 'ordinal_rec.vo' failed
make[2]: *** [ordinal_rec.vo] Error 1
make[2]: *** Waiting for unfinished jobs....
```
that seems like some sort of build race in Equations.

--- a/examples/Makefile
+++ b/examples/Makefile
@@ -19,6 +19,7 @@ $(vfiles:.v=.vo): %.vo: %.v
 	$(COQC) $(COQFLAGS) $<
 
 graph_complete.vo: HoTT_light.vo
+ordinal_rec.vo: ordinals.vo
 
 $(vfiles:.v=.tex): %.tex: %.vo
 	$(COQDOC) --latex --interpolate --parse-comments ${<:.vo=.v}

